.cirrus.yml

```diff
@@ -28,12 +28,14 @@ env:
     #### Comment out fedora-35 for podman 4.x branches.
     ####
     FEDORA_NAME: "fedora-36"
+    FEDORA_AARCH64_NAME: "${FEDORA_NAME} {aarch64}"
```

Comment:
Request: `-` (dash) instead of space? I suppose I can live with the curly braces, but I can't live with whitespace. Thanks.

Reply:
i'm cool with it but I'll defer to @cevich.

Reply:
Yes please, what Ed said, not having whitespace is probably important.